source/sass/components/nav.scss

```diff
@@ -76,11 +75,48 @@
 
     .nav-links {
       a {
+        $border-thickness: 6px;
+
+        border-bottom: 6px solid transparent;
+        color: $black;
+        font-family: "Nunito Sans";
+        font-weight: 700;
         margin-right: 18px;
+        transition: border-color 0.25s ease;
+
+        @mixin padding-bottom($bp: sm) {
+          padding-bottom: calc(#{map-get($menu-container-padding, $bp)} + #{map-get($primary-nav-link-padding, $bp)} - #{$border-thickness} / 2);
+        }
+
+        @media (min-width: $bp-sm) {
+          @include padding-bottom(sm);
+        }
+
+        @media (min-width: $bp-md) {
+          @include padding-bottom(md);
+        }
+
+        @include scaleText(17px, 23px);
+
+        @include hover-focus-active {
+          text-decoration: none;
+          border-bottom-color: #ccc;
+        }
       }
 
-      @media (max-width: $bp-lg) {
-        padding-top: 0.6rem;
+      @at-root {
+        #view-people .nav-link-people,
+        #view-initiatives .nav-link-initiatives,
+        #view-projects .nav-link-projects,
+        #view-participate .nav-link-participate,
+        #view-internet-health .nav-link-internet-health,
+        #view-about .nav-link-about {
+          border-color: black !important;
```

Comment:
This whole section is weird (though I know it's been here). The `important`s feel really wrong though. Can we remove them if we make line 76 not as nested, and make it so the selector is `#primary-nav-container [class^="nav-link-"]`?

Reply:
@alanmoo I'll create a followup ticket to do this. There are too many elements using the class `nav-link` so I don't wanna tackle this in this PR just yet. 

I can include this work as part of https://github.com/mozilla/foundation.mozilla.org/pull/2348 or work on it after https://github.com/mozilla/foundation.mozilla.org/pull/2348 is merged.

Reply:
Follow-up tickets created: https://github.com/mozilla/foundation.mozilla.org/issues/2360